### DISCUSSION TITLE
Clustering in Multiome dataset

### DISCUSSION THREAD

**Discussion by sylestiel** [stuart-lab/signac#1147] [Q&A] [answered]
In the absence of predicted ids based on transfer anchors how can one generate clusters in Multiome dataset?
In other words, can we run FindClusters() in lieu of applying predicted ids to the embedding? If so, what assay do we specify?

**Comment by timoast** [comment, marked as answer]
Yes you can, you can use either scATAC-seq or scRNA-seq assays, or use a weighted combination of them using the WNN method as shown in the multiome vignette.
